xrm/xrm.d.ts

```diff
@@ -344,6 +349,9 @@ declare namespace Xrm
         prependOrgName( sPath: string ): string;
     }
 
+    export module Context {
```

Comment:
Just in case there would be another Theme domain in the future.  Could also name it ContextTheme as well I guess.  I'm fine removing it as well.  Just let me know and I'll make the needed changes.